src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultDisplayTemplates.cs

```diff
@@ -286,12 +287,13 @@ private static bool ShouldShow(ModelExplorer modelExplorer, TemplateInfo templat
                 !templateInfo.Visited(modelExplorer);
         }
 
-        public static string StringTemplate(IHtmlHelper htmlHelper)
+        public static IHtmlContent StringTemplate(IHtmlHelper htmlHelper)
         {
-            return htmlHelper.Encode(htmlHelper.ViewData.TemplateInfo.FormattedModelValue);
+            var value = htmlHelper.ViewData.TemplateInfo.FormattedModelValue;
+            return new StringHtmlContent(value == null ? string.Empty : value.ToString());
```

Comment:
Suggest moving the ternary outside the constructor i.e. `return value == null ? HtmlString.Empty : new StringHtmlContent(value.ToString());`